CHANGELOG.md

```diff
@@ -23,6 +23,9 @@ and this project adheres to
 
 ### Fixed
 
+- [#4916](https://github.com/firecracker-microvm/firecracker/pull/4916): Fix
+  `IovDeque` implementation to work with any host page size.
```

Comment:
Not sure it can be called a bug when we explicitly designed for 4K pages at the implementation time. Also as I understand, if no net devices were used, 16K pages should work fine.